tests/app/main/views/test_service_updates.py

```diff
@@ -24,105 +24,80 @@ def test_should_render_activity_page_with_date(self, data_api_client):
-                '<a href="#example-textbox" class="validation-masthead-link"><label for="acknowledged">acknowledged</label></a>'),  # noqa
-            self._replace_whitespace(response.get_data(as_text=True))
-        )
-        data_api_client.find_audit_events.assert_not_called()
+        assert self._replace_whitespace(
+            '<a href="#example-textbox" class="validation-masthead-link"><label for="acknowledged">acknowledged' +
+            '</label></a>'
+        ) in self._replace_whitespace(response.get_data(as_text=True))
+        assert data_api_client.find_audit_events.called is False
 
     @mock.patch('app.main.views.service_updates.data_api_client')
     def test_should_allow_valid_submission_all(self, data_api_client):
         data_api_client.find_audit_events.return_value = {'auditEvents': [], 'links': {}}
 
         response = self.client.get('/admin/service-updates?audit_date=2006-01-01&acknowledged=all')
-        self.assertEquals(200, response.status_code)
-        self.assertIn(
-            '<input class="filter-field-text" id="audit_date" name="audit_date" placeholder="eg, 2015-07-23" type="text" value="2006-01-01">',  # noqa
-            response.get_data(as_text=True)
-        )
+        assert response.status_code == 200
+        assert '<input class="filter-field-text" id="audit_date" name="audit_date" placeholder="eg, 2015-07-23" type='\
+            '"text" value="2006-01-01">' in response.get_data(as_text=True)
 
-        self.assertIn(
-            self._replace_whitespace(
-                '<inputname="acknowledged"value="all"id="acknowledged-1"type="radio"aria-controls=""checked>'),  # noqa
-            self._replace_whitespace(response.get_data(as_text=True))
-        )
-        data_api_client.find_audit_events.assert_called()
+        assert self._replace_whitespace(
+            '<inputname="acknowledged"value="all"id="acknowledged-1"type="radio"aria-controls=""checked>'
+        ) in self._replace_whitespace(response.get_data(as_text=True))
+        assert data_api_client.find_audit_events.called
 
     @mock.patch('app.main.views.service_updates.data_api_client')
     def test_should_allow_valid_submission_date_fields(self, data_api_client):
         data_api_client.find_audit_events.return_value = {'auditEvents': [], 'links': {}}
 
         response = self.client.get('/admin/service-updates?audit_date=2006-01-01')  # noqa
-        self.assertEquals(200, response.status_code)
-        self.assertIn(
-            '<input class="filter-field-text" id="audit_date" name="audit_date" placeholder="eg, 2015-07-23" type="text" value="2006-01-01">',  # noqa
-            response.get_data(as_text=True)
-        )
+        assert response.status_code == 200
+        assert '<input class="filter-field-text" id="audit_date" name="audit_date" placeholder="eg, 2015-07-23" type='\
+            '"text" value="2006-01-01">' in response.get_data(as_text=True)
+
+        assert self._replace_whitespace(
```

Comment:
The call to `_replace_whitespace` seems unnecessary here (and some other places later on) as there's clearly no whitespace in the string.  Keeps the comparison nice and symmetrical though I guess.